### PR TITLE
Fix rendering graph & instructions in black

### PR DESCRIPTION
**Pull request by hearsilent**
Fix issue #594 

## Before
device-2018-02-21-172357
device-2018-02-21-172342

## After
device-2018-02-21-174436
device-2018-02-21-165658

### CODE REVIEW

**Comment by hearsilent** [comment]
@gpeal Please review this, thx 💯

File: LottieSample/build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ apply plugin: 'kotlin-android-extensions'
 
 android {
   compileSdkVersion 27
-  buildToolsVersion "27.0.2"
+  buildToolsVersion "27.0.3"
```
Comment (gpeal) [inline LottieSample/build.gradle]: This should really be done as a separate PR

Comment (hearsilent) [reply LottieSample/build.gradle]: Got it, thanks for your suggestion.
